netbox/users/admin/__init__.py

```diff
@@ -58,9 +58,15 @@ def get_inlines(self, request, obj):
 class TokenAdmin(admin.ModelAdmin):
     form = forms.TokenAdminForm
     list_display = [
-        'key', 'user', 'created', 'expires', 'write_enabled', 'description'
+        'key', 'user', 'created', 'expires', 'write_enabled', 'description', 'list_allowed_ips'
     ]
 
+    def list_allowed_ips(self, obj):
+        if obj.allowed_ips:
+            return obj.allowed_ips
+        return 'Any'
```

Comment:
```suggestion
        return obj.allowed_ips or 'Any'
```

Reply:
Nice ! Whish I knew how to code this clean :-)